packages/frontend-2/components/viewer/explorer/Explorer.vue

```diff
@@ -89,7 +90,14 @@ const rootNodes = computed(() => {
 
   if (!worldTree.value) return []
   expandLevel.value = -1
-  const nodes = []
+
+  // Arrays to hold categorized nodes
+  const versionNodes = []
```

Comment:
@andrewwallacespeckle I'm wondering if there's a way to avoid duplicating this sorting logic. Since `resourceItems` is already sorted correctly, maybe we can just colocate each node of rootNodes with its resourceItem and use its idx in that array to set up the order?

```ts
const results: Record<number, ExplorerNode> = {}
for (const node of rootNodes) {
  const objectId = ...
  const order = resourceItems.value.findIndex(ri => ri.objectId === objectId)
  results[order] = node
}

return sortBy(Object.values(results), i => i[0]).map(i => i[1])
```